features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Applications/ApplicationFormHanlder.jsx

```diff
@@ -378,47 +420,49 @@ class NewApp extends React.Component {
                             handleDeleteChip={this.handleDeleteChip}
                             handleAddChip={this.handleAddChip}
                         />
-                    </div>
-                    <div className={classes.buttonWrapper}>
-                        <Button variant='outlined' className={classes.button} onClick={handleClose}>
-                            <FormattedMessage
-                                id='Applications.Create.NewApp.cancel'
-                                defaultMessage='Cancel'
-                            />
-                        </Button>
-                        <Button
-                            variant='contained'
-                            color='primary'
-                            className={classes.button}
-                            onClick={this.saveApplication}
-                        >
-                            <FormattedMessage
-                                id='Applications.Create.NewApp.add.new.application.button'
-                                defaultMessage='ADD NEW APPLICATION'
-                            />
-                        </Button>
-                    </div>
-                </Dialog>
-            </React.Fragment>
+                        <Box display='flex' justifyContent='flex-start' mt={4} spacing={1}>
+                            <Box>
+                                <Button
+                                    variant='contained'
+                                    color='primary'
+                                    onClick={isEdit ? this.saveEdit : this.saveApplication}
+                                >
+                                    <FormattedMessage
+                                        id='Applications.Create.ApplicationFormHanlder.save'
+                                        defaultMessage='save'
```

Comment:
Shouldn't this be "Save"?

Reply:
fixed in https://github.com/wso2/carbon-apimgt/pull/7418/commits/a76567b6141be3484b2cf75e27a28dab9242c0fd

everything was capitalized to match publisher buttons